tests/unit/customizations/codeartifact/test_adapter_login.py

```diff
@@ -52,7 +52,7 @@ def test_get_commands(self):
             )
 
     def test_run_commands_nonexistent_command(self):
```

Comment:
We should make sure to add a test case that output from stderr of the subprocess shows up in the stderr propagated to the user.

Reply:
Added a test